R/common-error-checking-utils.R

```diff
@@ -0,0 +1,24 @@
+#' @title 
```

Comment:
We collaboratively decided not to include the warning for the function since it mucks up the output of `devtools::test()`.